Source/ApiTemplate/Source/ApiTemplate/CustomServiceCollectionExtensions.cs

```diff
@@ -22,7 +22,7 @@ namespace ApiTemplate
     using Boxed.AspNetCore.Swagger.SchemaFilters;
 #endif
     using Microsoft.AspNetCore.Builder;
-#if (!ForwardedHeaders && HostFiltering)
```

Comment:
Please ignore these warnings about brackets. These `#if` pre-processor directives are used for optionally selecting features. Unfortunately, there is a bug in `dotnet new` which stops it from working without these brackets.

Reply:
I reverted those changes